### PR TITLE
docs: fix code examples style in build doc

### PR DESCRIPTION
**Pull request by alan-agius4**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Code examples style is not consistent 

## What is the new behavior?
fix code examples styles

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/build.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ For example:
 
 The base file `environment.ts`, contains the default environment settings. For example:
 
-<code-example language="none" class="code-shell">
+<code-example language="none">
```
Comment (jenniferfell) [inline aio/content/guide/build.md]: Good catch, but I think this should actually be the style of file contents, not command line input. The original had more than one problem. See the examples immediate following this.
```
export const environment = {	export const environment = {
  production: false	  production: false
};	};
```

File: aio/content/guide/build.md [hunk 2]
```diff
@@ -144,7 +144,11 @@ You can add additional configurations as required. To add a staging environment,
 You can add more configuration options to this target environment as well. 
 Any option that your build supports can be overridden in a build target configuration.
 
-To build using the staging configuration, run `ng build --configuration=staging`.
+To build using the staging configuration, run
+
+<code-example language="none" class="code-shell">
```
Comment (jenniferfell) [inline aio/content/guide/build.md]: This should have `language=sh` as in 
```
<code-example language="sh" class="code-shell">
```
per https://angular.io/guide/docs-style-guide

File: aio/content/guide/build.md [hunk 3]
```diff
@@ -144,7 +144,11 @@ You can add additional configurations as required. To add a staging environment,
 You can add more configuration options to this target environment as well. 
 Any option that your build supports can be overridden in a build target configuration.
 
-To build using the staging configuration, run `ng build --configuration=staging`.
+To build using the staging configuration, run
```
Comment (jenniferfell) [inline aio/content/guide/build.md]: Please change to: To build using the staging configuration, run the following command:

**Review by jenniferfell — CHANGES_REQUESTED** [review]
Thanks for finding these and submitting fixes. A few small requests. Thanks!

**Comment by alan-agius4** [comment]
@jenniferfell, updated.

**Review by brandonroberts — APPROVED** [review]

**Comment by brandonroberts** [comment]
@alan-agius4 will you rebase this branch so the tests will run again?

**Comment by alan-agius4** [comment]
@brandonroberts, done.

**Review by jenniferfell — APPROVED** [review]
Thanks for making those changes. LGTM

**Pull request closed without merging**